File: DK07/DK07.cpp
#include<bits/stdc++.h>
using namespace std;

int main() {
    int a, b, c;
    cin >> a >> b >> c;
    if (a == 0) {
        if (b == 0) {
            if (c == 0) {
                cout << "WOW" << endl;
            } else {
                cout << "NO" << endl;
            }
        } else {
            double x = -(double)c / b;
            cout << fixed << setprecision(2) << x << endl;
        }
    } else {
        double delta = b * b - 4.0 * a * c;
        if (delta < 0) {
            cout << "NO" << endl;
        } else if (delta == 0) {
            double x = -(double)b / (2 * a);
            cout << fixed << setprecision(2) << x << endl;
        } else {
            double sqrt_delta = sqrt(delta);
            double x1 = (-b - sqrt_delta) / (2.0 * a);
            double x2 = (-b + sqrt_delta) / (2.0 * a);
            if (x1 > x2) {
                swap(x1, x2);
            }
            cout << fixed << setprecision(2) << x1 << " " << x2 << endl;
        }
    }
    return 0;
}

File: DK07/DK07.py
import math

a, b, c = map(int, input().split())

if a == 0:
    if b == 0:
        if c == 0:
            print("WOW")
        else:
            print("NO")
    else:
        x = -c / b
        print(f"{x:.2f}")
else:
    delta = b*b - 4*a*c
    if delta < 0:
        print("NO")
    elif delta == 0:
        x = -b / (2*a)
        print(f"{x:.2f}")
    else:
        sqrt_delta = math.sqrt(delta)
        x1 = (-b - sqrt_delta) / (2*a)
        x2 = (-b + sqrt_delta) / (2*a)
        if x1 > x2:
            x1, x2 = x2, x1

        print(f"{x1:.2f} {x2:.2f}")
